docs/reference/operator_parameters.md

```diff
@@ -221,6 +221,9 @@ CRD-based configuration.
   memory limits for the postgres containers, unless overridden by cluster-specific
   settings. The default is `1Gi`.
 
+* **set_memory_request_to_limit**
+  Set `memory_request` to `memory_limit` for all Postgres clusters (the default value is also increased). This prevents certain cases of memory overcommitment at the cost of overprovisioning memory and potential scheduling problems for Postgres pods with high memory limits due to the lack of memory on Kubernetes cluster nodes. This affects sidecar containers as well. The default is `false`.
```

Comment:
This affects all possible containers, not only Postgres, might need a word on that.